shanghai/connection.py

```diff
@@ -38,7 +39,7 @@ def close(self):
             while not reader.at_eof():
                 line = await reader.readline()
                 line = line.strip()
-                current_logger.debug(">", line)
+                current_logger.info(">", line)
```

Comment:
well, if the bot chokes, you'd have to .. debug, right? :hamster: 

But well, either way is fine with me anyway.